Makefile

```diff
@@ -89,35 +89,36 @@ test: checkstyle
 endif
 
 .PHONY: travis
-travis:
-	if test "x$$TRAVIS" != "xtrue"; then \
-	  echo "You shouldn't run this outside of travis-ci env" ;\
-	  exit 1 ;\
-	fi ;\
-	export MOJO_LOG_LEVEL=debug ;\
-	export MOJO_TMPDIR=$$(mktemp -d) ;\
-	export OPENQA_LOGFILE=/tmp/openqa-debug.log ;\
-	export TEST_PG=DBI:Pg:dbname=openqa_test ;\
+docker-tests:
 	if test "x$$FULLSTACK" = x1 || test "x$$SCHEDULER_FULLSTACK" = x1; then \
-		git clone https://github.com/os-autoinst/os-autoinst.git ../os-autoinst ;\
-		(cd ../os-autoinst && SETUP_FOR_TRAVIS=1 sh autogen.sh && make ) ;\
-					eval $$(dbus-launch --sh-syntax) ;\
-		export PERL5OPT="$$PERL5OPT $$HARNESS_PERL_SWITCHES"; \
+		git clone -b 35383 https://github.com/dasantiago/os-autoinst.git ../os-autoinst ;\
```

Comment:
what's different in there?

Reply:
forgot to remove that. The autogen.sh in os-autoinst is different.

Need to create a PR for that :-)